app/_how-tos/kic-fallback-config-exclude.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Fallback configuration: Exclude broken objects"
+description: |
+  Remove broken configuration from your {{ site.base_gateway }} configuration automatically
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/fallback-configuration/exclude/
+
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - /kubernetes-ingress-controller/fallback-configuration/
+
+products:
+  - kic
+
+tools:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+entities: []
+
+tldr:
+  q: How do I exclude a broken subset of my {{ site.kic_product_name }} configuration?
+  a: Enable the `FallbackConfiguration` feature gate for {{ site.kic_product_name }}
+
+prereqs:
+  kubernetes:
+    gateway_api: true
+    feature_gates: FallbackConfiguration=true
+    dump_config: true
+  entities:
+    services:
+      - echo-service
+
+cleanup:
+  inline:
+    - title: Uninstall KIC from your cluster
+      include_content: cleanup/products/kic
+      icon_url: /assets/icons/kubernetes.svg
+
+related_resources:
+  - text: Fallback configuration overview
+    url: /kubernetes-ingress-controller/fallback-configuration/
+  - text: "How-To: Backfill broken configuration"
+    url: /kubernetes-ingress-controller/fallback-configuration/backfill/
+---
+
+## Scenario
+
+In this example, we'll consider a situation where:
+
+1. We have two Routes pointing to the same `Service`. One Route is configured with `KongPlugin`s providing authentication and base rate-limiting. Everything works as expected.
+2. We add one more rate-limiting `KongPlugin` that is to be associated with the second Route and a specific `KongConsumer` so that it can be rate-limited in a different way than the base rate-limiting, but we forget associate the `KongConsumer` with the `KongPlugin`. It results in the Route being broken because of duplicated rate-limiting plugins.
```

Comment:
```suggestion
2. We add one more rate limiting `KongPlugin` that will be associated with the second Route and a specific `KongConsumer` so that it can be rate limited in a different way than the base rate limiting. But, we forget to associate the `KongConsumer` with the `KongPlugin`. It results in the Route being broken because of duplicated rate limiting plugins.
```